src/main/java/com/eternalcode/discordapp/expierience/ExpierienceListener.java

```diff
@@ -0,0 +1,52 @@
+package com.eternalcode.discordapp.expierience;
+
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import java.util.concurrent.ExecutionException;
+
+public class ExpierienceListener extends ListenerAdapter {
+    private final ExperienceRepository experienceRepository;
+    private static final int HOW_MANY_WORDS_TO_GIVE_POINTS = 5;
```

Comment:
Ta wartość powinna być konfigurowalna, proszę nie używać magic numbers.

Reply:
To nie jest magic word pozdrawiam

Reply:
> Ta wartość powinna być konfigurowalna, proszę nie używać magic numbers.

? Why not lol

Reply:
W jakim celu mamy przeprowadzać przegląd obecnego systemu XP? Jeśli już oczekujesz tego typu działań, to najlepiej pozostawić system XP w swojej ostatecznej postaci. Jeśli masz na myśli zastąpienie go nowszym systemem bez zachowania tego do którego mamy zrobić review, to jesteś w wielkim błędzie. Nie pozwolę na to, ponieważ w takim przypadku wprowadzisz kod, który nigdzie nie zostanie wykorzystany. Jestem przekonana, że kontynuowanie pracy w oparciu o istniejący system jest najlepszym rozwiązaniem, które zapewni stabilność i bezpieczeństwo systemu.